src/net/mod.rs

```diff
@@ -16,6 +16,8 @@ mod types;
 #[cfg(windows)]
 mod wsa;
 
+#[cfg(target_os = "linux")]
```

Comment:
Is this API known to be unavailable on Android? If not, would it make sense to change this from `target_os = "linux"` to `linux_kernel`, which includes "linux" and "android"?

Reply:
I looked into the bionic source and it seems to be available. I changed the cfgs accordingly.